hls4ml/templates/vivado/nnet_utils/nnet_conv1d_stream.h

```diff
@@ -1,34 +1,30 @@
 #ifndef NNET_CONV1D_STREAM_H_
 #define NNET_CONV1D_STREAM_H_
 
+#include "hls_stream.h"
 #include "nnet_common.h"
 #include "nnet_conv_stream.h"
-#include "hls_stream.h"
 
 namespace nnet {
 
-template<class data_T, typename CONFIG_T>
-void compute_scaled_indices_1d(
-    const unsigned w_idx,
-    ap_uint<CONFIG_T::filt_width> *pixel_idx
-) {
+template <class data_T, typename CONFIG_T>
+void compute_scaled_indices_1d(const unsigned w_idx, ap_uint<CONFIG_T::filt_width> *pixel_idx) {
     unsigned wp_idx = w_idx * (data_T::size / CONFIG_T::n_chan);
 
-    ComputeIndex: for (unsigned p = 0; p < data_T::size / CONFIG_T::n_chan; p++) {
+ComputeIndex:
```

Comment:
@jmduarte Any idea why the code formatter decided to use one level less indented for the loop labels? Looks a bit ugly. Can we set the indentation to be the same level as the loop?

Reply:
@jmduarte is the expert on the formatting. All of it was automatically done.

Reply:
Maybe that's the style, where the labels are one tab stop to the left? If it's consistent, it's not too bad.